packages/workflow/src/Extensions/NumberExtensions.ts

```diff
@@ -86,56 +94,106 @@ function toDateTime(value: number, extraArgs: [DateTimeFormat]) {
+		{ example: '(42).isOdd()', evaluated: 'false' },
+	],
 	returnType: 'boolean',
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-isOdd',
 };
 
 format.doc = {
 	name: 'format',
 	description:
-		'Returns a formatted string of a number based on the given `LanguageCode` and `FormatOptions`. When no arguments are given, transforms the number in a format like `1.234`.',
+		'Returns a formatted string representing the number. Useful for formatting for a specific language or currency. The same as <a target="_blank" href=”https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat”><code>Intl.NumberFormat()</code></a>.',
+	examples: [
+		{ example: "(123456.789).format('de-DE')", evaluated: '123.456,789' },
+		{
+			example: "(123456.789).format('de-DE', {'style': 'currency', 'currency': 'EUR'})",
+			evaluated: '123.456,79 €',
+		},
+	],
 	returnType: 'string',
 	args: [
-		{ name: 'locales?', type: 'LanguageCode' },
-		{ name: 'options?', type: 'FormatOptions' },
+		{
+			name: 'locale',
+			optional: true,
+			description:
+				'A <a target="_blank" href=”https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locales_argument”>locale tag</a> for formatting the number, e.g. <code>fr-FR</code>, <code>en-GB</code>, <code>pr-BR</code>',
+			default: '"en-US"',
+			type: 'string',
+		},
+		{
+			name: 'options',
+			optional: true,
+			description:
+				'Configuration options for number formatting. <a target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat" target="_blank">More info</a>',
+			type: 'object',
+		},
 	],
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-format',
 };
 
 round.doc = {
 	name: 'round',
 	description:
-		'Returns the value of a number rounded to the nearest whole number, unless a decimal place is specified. Defaults to 0 decimal places if no argument is given.',
+		'Returns the number rounded to the nearest whole number (or specified number of decimal places)',
+	examples: [
+		{ example: '(1.256).round()', evaluated: '1' },
+		{ example: '(1.256).round(1)', evaluated: '1.3' },
+		{ example: '(1.256).round(2)', evaluated: '1.26' },
+	],
 	returnType: 'number',
-	args: [{ name: 'decimalPlaces?', type: 'number' }],
+	args: [
+		{
+			name: 'decimalPlaces',
+			optional: true,
+			description: 'The number of decimal places to round to',
+			default: '0',
+			type: 'number',
+		},
+	],
 	docURL: 'https://docs.n8n.io/code/builtin/data-transformation-functions/numbers/#number-round',
 };
 
 toBoolean.doc = {
 	name: 'toBoolean',
-	description: 'Converts a number to a boolean. 0 is `false`, all other numbers are `true`.',
+	description:
+		'Converts the number to a boolean value. <code>0</code> becomes <code>false</code>; everything else becomes <code>true</code>.',
```

Comment:
1. Wordy?

```suggestion
		'Returns <code>false</code> for <code>0</code> and <code>true</code> for any other number',
```

2. Here we're using `<code>` tags for numbers, but not in `Boolean.toNumber`.

3. What about negative numbers? A user might reasonably expect a negative number to be `0`.

Reply:
3. Added clarification to the description